app/src/main/res/layout/bsd_open_channel.xml

```diff
@@ -124,6 +124,15 @@
 
         </androidx.constraintlayout.widget.ConstraintLayout>
 
+        <!-- Dummy item to prevent EditText from receiving focus -->
+        <LinearLayout
```

Comment:
On my emulators, the soft keyboard would pop up in the Open Channel View.
Could not reproduce with an actual device, but moving this fixed the issue on the emulators.